plugin-infra/plugin-metadata-store/src/com/thoughtworks/go/plugin/access/authorization/AuthorizationMetadataStore.java

```diff
@@ -56,4 +56,8 @@ public static AuthorizationMetadataStore instance() {
         }
         return plugins;
     }
+
+    public Set<String> getPluginsThatSupportsWebBasedAuthentication() {
```

Comment:
Should we consider returning the PluginInfo objects instead of just the set of pluginIds